test/benchmarks/driver_bench/readme.md

```diff
@@ -0,0 +1,48 @@
+# Node.js Driver Benchmarks
+
+Set up the driver for development (`npm ci` in the top level of this repo).
+
+Then:
```

Comment:
We also need a cluster running, and docs on how to configure the tests with env variables would be nice too

Reply:
agreed, added!